qiskit/circuit/library/standard_gates/ecr.py

```diff
@@ -21,7 +21,7 @@
 
 
 class ECRGate(Gate):
-    r"""An echoed RZX(pi/2) gate implemented using RZX(pi/4) and RZX(-pi/4).
+    r"""An echoed cross-resonance gate.
 
     This gate is maximally entangling and is equivalent to a CNOT up to
```

Comment:
Can you also update the doc string to include that the gate implements the unitary `\frac{1}{\sqrt}{IX-XY}` so its more clear.

Reply:
done